syncano_cli/execute/utils.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import json
+
+from syncano_cli.logger import get_logger
+
+LOG = get_logger('syncano-execute')
+
+
+def print_response(response):
```

Comment:
Consider this - this is a script code:
```
import json

set_response(HttpResponse(status_code=200, content=json.dumps({'one': 1}), content_type='application/json'))

print(12)
```

This:
```
s = Script.please.get(id=1)

t = s.run()

t.reload()
print(t._raw_data)
```

Will output:
```
{
    'status': u'success', 
    'links': <syncano.models.fields.LinksWrapper object at 0x7fd2dd91d690>, 
    'executed_at': datetime.datetime(2016, 6, 13, 12, 50, 23, 99456), 
    'instance_name': 'hidden-lake-9362', 
    'result': {
        u'response': {
            u'status': 200, 
            u'content': u'{"one": 1}', 
            u'content_type': u'application/json'}, 
        u'stderr': u'', 
        u'stdout': u'12'}, 
    'duration': 38, 
    'script_id': 1, 
    'id': 93
}

```
I can't see in your solution handling of the custom response - maybe I've missed something?